packages/frontend/src/features/catalog/components/CatalogPanel.tsx

```diff
@@ -385,33 +472,176 @@ export const CatalogPanel: FC = () => {
+                                                },
+                                            }}
+                                        >
+                                            Dimensions
+                                        </Badge>
+                                    }
+                                />
+
+                                <Checkbox
+                                    checked={filters.metrics}
+                                    onChange={() => {
+                                        toggleFilter(FilterType.Metrics);
+                                    }}
+                                    label={
+                                        <Badge
+                                            fw={500}
+                                            color="orange"
+                                            styles={{
+                                                root: {
+                                                    textTransform: 'none',
+                                                },
+                                            }}
+                                        >
+                                            Metrics
+                                        </Badge>
+                                    }
+                                />
+                            </Stack>
+
+                            <Divider c="gray.1" />
+                            <Stack spacing="xs">
+                                {/* 
+                                TODO: Don't know how this is supposed to work yet
+                                <Checkbox
+                                    checked={filters.hideBaseTables}
+                                    onChange={() => {
+                                        toggleFilter(FilterType.HideBaseTables);
+                                    }}
+                                    label={
+                                        <Text fz="xs" fw={500} c="gray.7">
+                                            Hide base tables
+                                        </Text>
+                                    }
+                                /> */}
+                                <Checkbox
+                                    checked={filters.hideGroupedTables}
+                                    onChange={() => {
+                                        toggleFilter(
+                                            FilterType.HideGroupedTables,
+                                        );
+                                    }}
+                                    label={
+                                        <Text fz="xs" fw={500} c="gray.7">
+                                            Hide grouped tables
+                                        </Text>
+                                    }
+                                />
+                            </Stack>
+                            <Divider c="gray.1" />
```

Comment:
I know it's in the designs but I would remove this - feel free to ignore - no change required tbh!

Reply:
Good catch